modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+// TODO
+// * networking/TBD
+// * networking/load-balancing-openstack.adoc
+// For thinking and reviewing, adding to networking/load-balancing-openstack.adoc
+
+[id="nw-osp-configuring-external-load-balancer_{context}"]
+= Configuring an external load balancer
+
+You can configure a {product-title} cluster on {rh-openstack-first} to use an external load balancer in place of the default load balancer.
+
+// Maybe an About mod in support
+
+.Prerequisites
+
+* On your load balancer, TCP over ports 6443, 443, and 80 must be available to any users of your system.
+
+** The {product-title} API uses port 6443. 
+
+** Cluster applications use ports 443 and 80.
+
+* On your load balancer, port 22623, which is used to serve ignition start-up configurations to nodes, is not exposed outside of the cluster.
+
+* Your load balancer must be able to access every machine in your cluster. Methods to allow this access include:
+** Creating the load balancer on a subnet that is in the cluster's network, and then attaching a router interface to the subnet from an `OpenShift-external-router` object.
```

Comment:
@iamemilio 

```suggestion
** Creating the load balancer on a subnet that is in the cluster's network, and then attaching the load balancer to the machines subnet.
```

Reply:
```suggestion
** Attaching the load balancer to the cluster's machine subnet.
```
In my opinion, this would be clearer

Reply:
you can create the node anywhere, as long as you attach it to the machine subnet. That is the important part, so I think its the only thing we should suggest.

Reply:
Groovy. :+1: